src/firefly/java/edu/caltech/ipac/firefly/server/visualize/fitseval/SpectralCubeEval.java

```diff
@@ -69,9 +69,9 @@ public static int findWaveTabHDU(BasicHDU[] HDUs, int hduIdx) {
                 String xtension= hIHeader.getStringValue("XTENSION");
                 String extName= hIHeader.getStringValue("EXTNAME");
                 if ("BINTABLE".equals(xtension) && waveHDUName.equals(extName)) {
-                    return tableIdx;
+                    return i;
```

Comment:
I am assuming firefly-467 will override this.

Reply:
Yes, in order to continue, I copied to implement the tab, I copied firefly-467 here.